pkg/controller/controller_test.go

```diff
@@ -1891,3 +1925,137 @@ func TestProvisionWithMountOptions(t *testing.T) {
 		t.Errorf("expected mount options %v; got: %v", expectedOptions, pv.Spec.MountOptions)
 	}
 }
+
+// TestProvisionFromPVC tests create volume from snapshot
+func TestProvisionFromPVC(t *testing.T) {
+	var requestedBytes int64 = 1000
+	var pvcName = "src-pvc"
+	//var timeNow = time.Now().UnixNano()
+	//var metaTimeNowUnix = &metav1.Time{
+	//		Time: time.Unix(0, timeNow),
+	//	}
```

Comment:
@j-griffith can we remove this commented code ?

Reply:
Oops, yes sorry!